teccloud-api/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "teccloud-api",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "nodemon --watch src --exec 'ts-node src/main.ts'",
```

¿Puede esto cambiarse o no?
```suggestion
    "dev": "nodemon --watch src --exec npm start",
```

---

Es buena pregunta, déjame lo pruebo